Services/UserService/Commands/EditUserCommand.cs

```diff
@@ -11,29 +16,70 @@ public class EditUserCommand : IEditUserCommand
     {
         private readonly IValidator<UserInfoRequest> userInfoValidator;
         private readonly IValidator<PasswordChangeRequest> passwordChangeValidator;
-        
+        private readonly IBus busControl;
+
         public EditUserCommand
             ([FromServices] IValidator<UserInfoRequest> userInfoValidator, 
-            [FromServices] IValidator<PasswordChangeRequest> passwordChangeValidator)
+            [FromServices] IValidator<PasswordChangeRequest> passwordChangeValidator,
+            [FromServices] IBus busControl)
         {
+            this.busControl = busControl;
             this.userInfoValidator = userInfoValidator;
             this.passwordChangeValidator = passwordChangeValidator;
         }
         
-        public Task<bool> Execute(EditUserRequest request)
+        public async Task<bool> Execute(EditUserRequest request)
         {
             var passwordRequest = request.PasswordRequest;
             var userInfo = request.UserInfo;
-            
+            PasswordHashChangeRequest passwordHashChangeRequest = null;
+
             if (!(string.IsNullOrEmpty(passwordRequest.OldPassword)
```

Comment:
We have validators for these. I understand that this is not your part of the code, but I would like to remove unnecessary checks.

Reply:
This is a necessary check, since not only the password can change.